code/modules/antagonists/heretic/knowledge/cosmic_lore.dm

```diff
@@ -131,8 +127,7 @@
 		The combo is reset after two seconds without making an attack, \
 		or if you attack someone already marked. If you combo more than four attacks you will recieve, \
 		a cosmic trail and increase your combo timer up to ten seconds."
-	gain_text = "As he ascended to be a watcher, he needed to gather knowledge. \
-		He started to draw it at his home."
+	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. \ The blades now glistened with fragmented power. I fell to the ground and wept at the beast's feet."
```

Comment:
You have to actually split the line if you put in a line splitter
```suggestion
	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. \ 
		The blades now glistened with fragmented power. I fell to the ground and wept at the beast's feet."
```
Not going through all of these but yeah

Reply:
Oh I'm stupid I thought the slash did that